### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ghost** [cvmanjoo/yify_grabber#1]
Amazing app man i wondering can you implement a way to create a folder for each movie then download image for each of those movies? and insert it into the movie files?

**Comment by cvmanjoo** [comment]
Yeah! We can do that. Can u help me out for that?

**Comment by ghost** [comment]
I'm sure i can but you do know that yify has V2 of api out?

**Comment by cvmanjoo** [comment]
Yes, I'm aware of that. Should implement shortly. Making time for that.
If possible Make changes and push it.
